code/style.css

```diff
@@ -1,7 +1,273 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+html {
+  background: rgb(201, 238, 220);
+  color: black;
+  font-family: 'PT Sans Narrow', sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+/****1. HEADER, LOGO, NAVIGATION***/
+.header {
+  background-color: rgb(142, 226, 185);
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  display: flex;
+  flex-direction: column;
+  width: 75px;
+  height: 100%;
+  align-items: flex-start;
+  margin: 5px;
+  border: 4px solid black;
+}
+
+.navigation {
+  display: none;
+}
+
+
+/****2. HERO IMG AND TEXT***/
+.hero {
+  position: relative;
+}
+
+.hero-text {
+  font-size: 32px;
+  position: absolute;
+  top: 30%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: white; 
+  -webkit-text-stroke: 1px black;
+}
```

Comment:
Position absolute is another way of positioning text on top of another element, it seems to be working fine (referring to you mentioning you had issues on mobile).